.github/workflows/frameworks.yml

```diff
@@ -120,6 +120,8 @@ jobs:
               - *runner
               - bentoml/fastai.py
               - bentoml/_internal/frameworks/fastai.py
+              - bentoml/_internal/frameworks/common/pytorch.py
+              - tests/integration/frameworks/test_frameworks.py
```

Comment:
fastai uses pytorch data container, hence it is needed to also track the changes from there